server/src/test/java/io/crate/integrationtests/TableSettingsTest.java

```diff
@@ -88,8 +88,8 @@ public void testSelectSettingsColumn() throws Exception {
 
     @Test
     public void testSetNonDynamicTableSetting() {
-        assertThrowsMatches(() -> execute("alter table settings_table set (\"soft_deletes.enabled\"='true')"),
-                     isSQLError(containsString("Can't update non dynamic settings [[index.soft_deletes.enabled]] for open indices"),
+        assertThrowsMatches(() -> execute("alter table settings_table set (codec = 'best_compression')"),
```

Comment:
I chose also non-final setting here as we have now different message when update final settings